Cargo.toml

```diff
@@ -58,7 +58,6 @@ members = [
     "sidechain/rpc-handler",
     "sidechain/sidechain-crate",
     "sidechain/state",
-    "sidechain/top-pool-executor",
```

Comment:
This crate was removed completely. Its functionality was either obsolete or duplicated in the `top-pool-author`.